.travis.yml

```diff
@@ -17,6 +19,7 @@ before_install:
   # Setup Node.js version-specific dependencies
   - "test $TRAVIS_NODE_VERSION != '0.8' || npm rm --save-dev istanbul"
   - "test $(echo $TRAVIS_NODE_VERSION | cut -d. -f1) -ge 4 || npm rm --save-dev eslint"
+  - "test -z $(echo $HTTP2_TEST) || npm install --only=dev https://github.com/sogaani/express.git#initial-support-http2 https://github.com/sogaani/supertest.git#http2"
```

Comment:
Please only install deps from npm.

Reply:
Express and this PR has circular dependency. Would you allow me to add http2 enabled express package like `express-http2` for npm?

Reply:
Express use [cookie-session](https://github.com/expressjs/cookie-session) for test, and cookie-session depend on cokkies.

Reply:
And the tests in this module won't work with the published version of Express?

Reply:
Yes.
I met following errors with the published version of Express.
```
  1) Express should set cookies:
     Uncaught TypeError: Cannot read property 'readable' of undefined
      at IncomingMessage._read (_http_incoming.js:104:19)
      at IncomingMessage.Readable.read (_stream_readable.js:442:10)
      at IncomingMessage.read (_http_incoming.js:96:15)
      at resume_ (_stream_readable.js:822:12)
      at _combinedTickCallback (internal/process/next_tick.js:138:11)
      at process._tickCallback (internal/process/next_tick.js:180:9)
```